es-ds-components/components/es-zip-code-form.vue

```diff
@@ -105,6 +109,11 @@ const handleSubmit = () => {
         }"
         v-bind="$attrs">
         <div class="d-flex flex-column">
+            <div v-if="contextMessage">
+                <p class="mb-25 font-size-75">
```

Comment:
Small thing: since our performance checks regularly report that our pages have an excessive number of DOM elements, that's something I have my eye on in general - it'd save one element without impacting the layout by removing the `<div>` here and putting the `v-if` onto the `<p>`.